netty/src/test/java/io/grpc/netty/NettyServerHandlerTest.java

```diff
@@ -196,6 +199,19 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
   }
 
   @Test
+  public void transportReadyDelayedUntilConnectionPreface() throws Exception {
+    initChannel(new GrpcHttp2ServerHeadersDecoder(GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE));
+
+    handler().handleProtocolNegotiationCompleted(Attributes.EMPTY);
+    verify(transportListener, never()).transportReady(any(Attributes.class));
+
```

Comment:
s/any/isA/ ?

Reply:
They're equivalent here. But isA would actually be worse, because if I had the wrong class I could miss callbacks that I wanted to trigger failure. I'm not trying to verify it is an Attributes; I just checking the method which happens to have certain arguments.

Reply:
Oh, this `verify` checks `never()`, I should have meant the second `verify`.

Reply:
Yeah, the second verify would fall into "I'm not trying to verify it is an Attributes".